wikimedia/wikipedia-ios
# Talk pages error state - no fallback

**Phabricator:** Partially https://phabricator.wikimedia.org/T312314

### Notes
* This PR adds an error view for talk pages when no cache is available,  other error cases will come in follow up PRs

### Test Steps
1. Go to a talk page that doesn't have a cached version
2. before loading, turn the internet connection off
3. Make sure the error view appears 
4. Turn on the internet again, click the Try Again button
5. The talk page should load normally (load items or the empty view)
6. Do the same on a page that has a cached version
7. The cached items should load normally, no error view
